### PR TITLE
Updated the examples for LTR Policies and LTR Backups

### PR DESCRIPTION
**Pull request by rebeccaxu-ms**
# ARM (Control Plane) API Specification Update Pull Request 

## Purpose of this PR

What's the purpose of this PR? Check all that apply. This is **mandatory**!

  - [ ] New API version. (Such PR should have been generated with [OpenAPI Hub](https://aka.ms/openapihub), per [this wiki doc](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).)
  - [ ] Update existing version for a new feature. (This is applicable only when you are revising a private preview API version.)
  - [ ] Update existing version to fix swagger quality issues in S360.
  - [x] Other, please clarify:
    - _Updated the example responses for the LTR policies and LTR backups commands_

## Due diligence checklist

To merge this PR, you **must** go through the following checklist and confirm you understood 
and followed the instructions by checking all the boxes:

- [x] I have reviewed the general guidance on the spec PR review process: https://aka.ms/specprreview.
- [x] I confirm this PR is modifying Azure Resource Manager (ARM) related specifications, and not data-plane related specifications.
- [x] I commit to follow the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy).
- [x] I have reviewed following [Resource Provider guidelines](https://aka.ms/rpguidelines), including
  [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and
  [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md) (estimated time: 4 hours).  
  I understand this is required before I can request review from an ARM API Review board.

### ARM API changes review

- If you want for the ARM team to review this PR, you must add the `ARMReview` label. 
- The automation may automatically add the `ARMReview` label, if appropriate.  
  If this happens, proceed according to guidance given in GitHub comments also added by the automation.

### Breaking change review

If you have any breaking changes as defined in the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy/), 
follow the process outlined in the [High-level Breaking Change Process doc](https://aka.ms/breakingchangesprocess#high-level-breaking-change-process).
      
## Getting help

- For guidance on fixing this PR CI check failures, see the hyperlinks provided in given failure 
  and https://aka.ms/ci-fix.
- For additional help, see https://aka.ms/azsdk/support/spectools.

### CODE REVIEW

File: specification/sql/resource-manager/Microsoft.Sql/preview/2023-02-01-preview/examples/LongTermRetentionPolicyCreateOrUpdate.json [hunk 1]
```diff
@@ -26,8 +25,7 @@
           "weeklyRetention": "P1M",
           "monthlyRetention": "P1Y",
           "yearlyRetention": "P5Y",
-          "weekOfYear": 5,
-          "makeBackupsImmutable": true
```
Comment (danikaZ) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2023-02-01-preview/examples/LongTermRetentionPolicyCreateOrUpdate.json:30]: pls keep "makeBackupsImmutable": true

File: specification/sql/resource-manager/Microsoft.Sql/preview/2023-02-01-preview/examples/LongTermRetentionPolicyGet.json [hunk 2]
```diff
@@ -17,8 +17,7 @@
           "weeklyRetention": "P1M",
           "monthlyRetention": "P1Y",
           "yearlyRetention": "P5Y",
-          "weekOfYear": 5,
-          "makeBackupsImmutable": true
```
Comment (danikaZ) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2023-02-01-preview/examples/LongTermRetentionPolicyGet.json:21]: pls keep "makeBackupsImmutable": false

image

File: specification/sql/resource-manager/Microsoft.Sql/preview/2023-02-01-preview/examples/LongTermRetentionPolicyListByDatabase.json [hunk 3]
```diff
@@ -18,8 +18,7 @@
               "weeklyRetention": "P1M",
               "monthlyRetention": "P1Y",
               "yearlyRetention": "P5Y",
-              "weekOfYear": 5,
-              "makeBackupsImmutable": true
```
Comment (danikaZ) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2023-02-01-preview/examples/LongTermRetentionPolicyListByDatabase.json:22]: Just realize I missed testing this one, but I would imagine the return value would be same as get policy, please keep  makeBackupsImmutable

**Comment by rebeccaxu-ms** [comment]
@danikaZ Sorry I'm not sure what happened and why makeBackupsImmutable but thank you for catching that. If you could take another look now I'd really appreciate it!

**Review by danikaZ — APPROVED** [review]
lg, thx!

**Review by ericshape — APPROVED** [review]
only example changes.

**Comment by kazrael2119** [comment]
this js breaking is not caused by this pr

**Comment by ericshape** [comment]
the breaking change of clients are not caused by this PR

**Pull request merged**